snippets/faz-donenfeld-neves_crypto.md

```diff
@@ -0,0 +1,70 @@
+* Authors: Armando Faz, Jason A. Donenfeld, Samuel Neves
+* Reason: side-channel resistance required + performance critical
+* Source: https://github.com/armfazh/rfc7748_precomputed
+* License: [BSD-3-clause](https://github.com/armfazh/rfc7748_precomputed/blob/master/LICENSE)
```

Comment:
Maybe add `* Language: C`, as the rest of the snippets in this repo are written in Rust.